psi29a/unsf
# build fixes and updates:

- actually use cmake `TEST_BIG_ENDIAN` result: replace the
  `__BYTE_ORDER == __LITTLE_ENDIAN` check in the source with
  `!defined(WORDS_BIGENDIAN)`.
- refresh strtok_r.h from the CC0 licensed PDClib library
  at https://github.com/DevSolar/pdclib.git
- add strtok_r() check to cmake and adjust sources for it.
- makefile adjustments.
